numpy/ma/core.py

```diff
@@ -4582,50 +4606,47 @@ def all(self, axis=None, out=None):
         True
 
         """
+        kwargs = {} if keepdims is np._NoValue else {'keepdims': keepdims}
+
         mask = _check_mask_axis(self._mask, axis)
```

Comment:
To make sure, what happens with keepdims + _check_mask_axis? I guess `__setmask__` uses fun flat assignment logic, so that things just don't matter?